cfgov/v1/management/commands/update_data_snapshot_values.py

```diff
@@ -0,0 +1,67 @@
+import ast
+
+from django.core.management.base import BaseCommand
+
+from v1.models.browse_page import BrowsePage
+from v1.tests.wagtail_pages.helpers import publish_changes
+
+
+class Command(BaseCommand):
+    help = 'Monthly updates to data snapshot values'
+
+    def add_arguments(self, parser):
+        """Adds all arguments to be processed."""
+        parser.add_argument(
```

Comment:
Since this is running in Jenkins, we were trying to do this without impacting the job's workspace. Storing a temporary file with the data is likely a more-robust approach, so we're planning to switch to something like the above.